### PR TITLE
Update REAME

### PR DESCRIPTION
**Pull request by sergeyklay**
@andresgutierrez @niden  Create please composer repo from current `composer.json`

* Added license
* Improved Composer config
* Update README
* Minor code improvements

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,32 +1,65 @@
...
+
+```sh
+$ php composer.phar install
+```
+
+Create `var/config/config.php` file (use `config.sample.php` as example).
+
+## License
+
+Phosphorum is open-sourced software licensed under the [New BSD License][6]. © Phalcon Framework Team and contributors
```
Comment (iGusev) [inline README.md]: > Phosphorum

**Comment by niden** [comment]
Thank you!

**Comment by sergeyklay** [comment]
@niden  `phalcon/blog` Still not available


```sh
$ composer require "phalcon/blog":"dev-master"

./composer.json has been created
Loading composer repositories with package information
Updating dependencies (including require-dev)
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - The requested package phalcon/blog could not be found in any version, there may be a typo in the package name.

Potential causes:
 - A typo in the package name
 - The package is not available in a stable-enough version according to your minimum-stability setting
   see <https://groups.google.com/d/topic/composer-dev/_g3ASeIFlrc/discussion> for more details.

Read <https://getcomposer.org/doc/articles/troubleshooting.md> for further common problems.

Installation failed, deleting ./composer.json.
```

**Comment by niden** [comment]
@sergeyklay It will be added shortly :)

**Pull request merged**